exporters/otlp/src/otlp_exporter.cc

```diff
@@ -28,8 +28,18 @@ void PopulateRequest(const nostd::span<std::unique_ptr<sdk::trace::Recordable>>
   for (auto &recordable : spans)
   {
     auto rec = std::unique_ptr<Recordable>(static_cast<Recordable *>(recordable.release()));
-    // TODO - Handle Resource
     *instrumentation_lib->add_spans() = std::move(rec->span());
+
+    if (!has_resource)
+    {
+      std::unique_ptr<proto::resource::v1::Resource> proto_resource{
+          new proto::resource::v1::Resource};
+
+      rec->PopulateProtoResource(proto_resource.get());
+
+      resource_span->set_allocated_resource(proto_resource.release());
```

Comment:
nit - Just wondering if line 35 - 40 can be replaced with single line:

``
resource_span->mutable_resource() = rec->ProtoResource();
``

using older implementation of `ProtoResource`.

Reply:
I chose `set_allocated_resource` which could avoid allocate the `Resource` twice or more, but yes, it look simpler with `mutable_resource`. Updated the PR to the older implementation of `ProtoResource` as this optimization may not deserve the extra complexity.